client/app/index.html

```diff
@@ -221,6 +222,8 @@ <h3 translate>About</h3>
             'liveblog.posts',
             'liveblog.blog',
             'liveblog.themes',
+            'liveblog.freetypes',
```

Comment:
this will be even more confusing IMHO, why main functionality is just in edit and not the crud one... etc.  short names for modules is fine